ui/lib/core/addon/components/secrets-engine-mount-config.hbs

```diff
@@ -0,0 +1,15 @@
+<div ...attributes>
+  <ToggleButton
+    @isOpen={{this.showConfig}}
+    @openLabel="Hide mount configuration"
+    @closedLabel="Show mount configuration"
+    @onClick={{fn (mut this.showConfig) (not this.showConfig)}}
+    class="is-block"
+    data-test-mount-config-toggle
+  />
+  {{#if this.showConfig}}
+    {{#each this.fields as |field|}}
+      <InfoTableRow @label={{field.label}} @value={{field.value}} data-test-mount-config-field={{field.label}} />
+    {{/each}}
+  {{/if}}
```

Comment:
I could imagine some secret engines (such as KV) may want to show some config values that are specific to its type. Not a blocker for this work, but perhaps a `{{yield}}` here, or something fancier within `this.fields` definition

Reply:
Yes that's a good point. I was considering that but then decided to leave it as is and it can be iterated on when the case arises. Thinking about it more now though and I don't think a yield would hurt so I'll go ahead and add that!